package.json

```
@@ -266,7 +266,7 @@
                 },
                 {
                     "command": "containerApps.openInPortal",
-                    "when": "view == containerApps && viewItem =~ /ManagedEnvironment|containerApp/",
+                    "when": "view == containerApps && viewItem =~ /managedEnvironment|containerApp/",
```

Do the regex flags work here? e.g. would this work?
```suggestion
                    "when": "view == containerApps && viewItem =~ /managedEnvironment|containerApp/i",
```

---

Actually, it does.  Neat.